rdbms/src/main/java/io/spine/server/storage/jdbc/aggregate/AggregateEventRecordTable.java

```diff
@@ -100,26 +101,33 @@ protected WriteQuery composeUpdateQuery(I id, AggregateEventRecord record) {
      */
     enum Column implements TableColumn {
 
-        id(ID),
-        aggregate(BLOB),
+        id,
```

Comment:
Please follow the [code style](https://google.github.io/styleguide/javaguide.html) for the enum member names.

Reply:
Done.